packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -1310,6 +1311,14 @@ class WebpackCLI implements IWebpackCLI {
       cli.colors = cli.createColors(color);
     });
 
+    this.program.option("--config-registered", "Disable interpret a config file.");
```

Comment:
I think `--config-registered` is more good than [`--config-register`](https://github.com/webpack/webpack-cli/issues/3195#issue-1196668000).

Reply:
We define options differently, see this for reference
https://github.com/webpack/webpack-cli/blob/7fe9e3b8e10042a381decf2d1daaf7275f1fca15/packages/webpack-cli/src/webpack-cli.ts#L968-L978

Reply:
Oh, sorry for misleading. I found the implement for global options, so added this new option to it. I'll change that.